x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/graph_grouped_node_preview_panel/components/grouped_item/parts/header_row.test.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { DOCUMENT_TYPE_ENTITY } from '@kbn/cloud-security-posture-common/schema/graph/v1';
+import { groupedItemClick$, __resetGroupedItemClickDedupe } from '../../../events';
+import { GROUPED_ITEM_TITLE_TEST_ID } from '../../../test_ids';
+import { HeaderRow } from './header_row';
+
+const flushMicrotasks = () => new Promise((r) => setTimeout(r, 0));
```

Comment:
can you document why it is needed?
What does it solve?

I haven't found any reference of this pattern so it worth adding some details